Makefile

```diff
@@ -18,7 +18,11 @@ install-deps:
 	git clone $(SNACTOR_URL) snactor
 	cd snactor && git checkout $(SNACTOR_BRANCH) && make install-deps build install
 	rm -fr snactor
-	for i in $$(find src -name 'Makefile'); do make -f $$i install-deps; done
+	ifeq($(ACTOR), )
+		for i in $$(find src -name 'Makefile'); do make -f $$i install-deps; done
+	else
+		python install_actor_deps.py $(ACTOR)
```

Comment:
IMHO we do not need any python script in order to do this. Seeing the implementation of it I believe it could be done with few shell commands. Did you have any particular reason why to go this way?

Reply:
Maybe, but do we want to keep adding stuff into this main Makefile? This can get messy pretty fast, especially if we keep coming with more and more features.

Reply:
well
for i in $(find src -type f -name 'Makefile'); do [[ "$(basename $(dirname echo $i))" == "augeas" ]] && echo "make -f $i install-deps"; done 
is not so big mess in my opinion. I know it is not solution of everything, but i feel it is way simpler than python script which does the same on another 30 lines :-)

Reply:
I know where are you coming from. But, I'm willing to bet some money that we are going to add more features in the future and this makefile will grow like hell.
Yes, it is more code, but we can more easily add warrning/error messages and stuff like that.